packages/ui/src/views/repo/pull-request/details/components/conversation/sections/pull-request-merge-section.tsx

```diff
@@ -49,145 +91,103 @@ const PullRequestMergeSection = ({
+                <Layout.Vertical className="ml-6">
+                  <p className="text-14 font-normal text-foreground-4">
+                    Use the&nbsp;
+                    <Button variant="link_accent" size="text" onClick={handleCommandLineClick} asChild>
+                      <span>command line</span>
+                    </Button>
+                    &nbsp;to resolve conflicts
                   </p>
-                  <ol>
-                    {stepMap.map((item, index) => {
-                      return stepInfo(item, index)
-                    })}
-                  </ol>
-                </div>
+                </Layout.Vertical>
               )}
-              <span className="text-14 text-foreground-2">
-                Conflicting files <span className="text-foreground-4">{`(${conflictingFiles?.length || 0})`}</span>
-              </span>
-
-              {!isEmpty(conflictingFiles) && (
-                <div className="mt-2">
-                  {conflictingFiles?.map((file, idx) => (
-                    <div className="flex items-center gap-x-2 py-1.5" key={`${file}-${idx}`}>
-                      <Icon className="text-icons-1" size={16} name="file" />
-                      <span className="text-14 text-foreground-1">{file}</span>
-                    </div>
+            </>
+          }
+        />
+        <PanelAccordionShowButton
+          isShowButton={isConflicted}
+          value={ACCORDION_VALUE}
+          accordionValues={accordionValues}
+        />
+      </Accordion.Trigger>
+      {isConflicted && (
+        <Accordion.Content className="ml-6">
+          <>
+            {showCommandLineInfo && (
+              <div className="mb-3.5 rounded-md border border-border p-1 px-4 py-2">
+                <h3 className="text-14 text-foreground-1">Resolve conflicts via command line</h3>
+                <p className="pb-4 pt-1 text-14 text-foreground-4">
+                  If the conflicts on this branch are too complex to resolve in the web editor, you can check it out via
+                  command line to resolve the conflicts
+                </p>
+                <ol className="flex flex-col gap-y-3">
+                  {stepMap.map((item, index) => (
+                    <StepInfo key={item.step} item={item} index={index} />
                   ))}
-                </div>
-              )}
-            </div>
-          </Accordion.Content>
-        )}
-      </Accordion.Item>
-    </Accordion.Root>
+                </ol>
+              </div>
+            )}
+            <span className="text-14 text-foreground-2">
+              Conflicting files <span className="text-foreground-4">{`(${conflictingFiles?.length || 0})`}</span>
+            </span>
+
+            {!isEmpty(conflictingFiles) && (
+              <div className="mt-1">
+                {conflictingFiles?.map((file, idx) => (
+                  <div className="flex items-center gap-x-2 py-1.5" key={`${file}-${idx}`}>
```

Comment:
I would've thought `file` would be unique
```suggestion
                {conflictingFiles?.map(file => (
                  <div className="flex items-center gap-x-2 py-1.5" key={file}>
```